ES/ESqs005.xml

```diff
@@ -603,6 +603,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="VP" when="2015-04-13">last edited</change>
          <change who="PL" when="2016-05-10">transformed from mycore to TEI P5</change>
          <change when="2019-04-01" who="ES">started adjusting</change>
+         <change who="DR" when="2021-12-10">Added IDs of owners</change>
```

Comment:
Thank you! To consider for [BLorient672](https://github.com/BetaMasaheft/Manuscripts/pull/1454#)?